src/main/java/org/jenkinsci/maven/plugins/hpi/AbstractJettyMojo.java

```diff
@@ -1,5 +1,5 @@
 //========================================================================
-//$Id: AbstractJettyMojo.java 36040 2010-10-18 11:24:56Z kohsuke $
+//$Id$
```

Comment:
The only change from upstream is the mojo annotations (javadoc -> java annotations)